Formula/anycloud.rb

```diff
@@ -0,0 +1,37 @@
+class Anycloud < Formula
+    desc "Lambda alternative that works with multiple cloud providers"
+    homepage "https://github.com/alantech/anycloud"
+    url "https://github.com/alantech/anycloud/archive/v0.1.2.tar.gz"
+    sha256 "3ca6c849aace024c75e5429db95b37e1a5592902557439996508b469cc706370"
```

Comment:
We can make updating these part of the Github Action that creates the archive itself in a followup PR.